CodeGeneration/Sources/SyntaxSupport/Child.swift

```diff
@@ -103,6 +111,22 @@ public class Child {
     return .identifier(lowercaseFirstWord(name: name))
   }
 
+  /// If this child has node choices, the type that the nested `SyntaxChildChoices` type should get.
+  ///
+  /// For any other kind of child nodes, accessing this property crashes.
+  public var syntaxChoicesType: TypeSyntax {
+    precondition(kind.isNodeChoices, "Cannot get `syntaxChoicesType` for node that doesn’t have nodeChoices")
+    return "\(raw: name.withFirstCharacterUppercased)"
+  }
+
+  /// If this child only has tokens, the type that the generated `TokenSpecSet` should get.
+  ///
+  /// For any other kind of child nodes, accessing this property crashes.
+  public var syntaxTokenType: TypeSyntax {
```

Comment:
I’m not sure if I suggested `syntaxTokenType` but in light of `syntaxChoicesType`, I think `tokenSpecSetType` would be a better name.

Also remember to update the `precondition` below.

Reply:
Thanks. Done